internal/image/oci_test.go

```diff
@@ -275,7 +275,7 @@ func TestManifestOCI1EmbeddedDockerReferenceConflicts(t *testing.T) {
 func TestManifestOCI1Inspect(t *testing.T) {
 	configJSON, err := os.ReadFile("fixtures/oci1-config.json")
 	require.NoError(t, err)
-
+	var annotations map[string]string
```

Comment:
It would be a bit easier to maintain if the three tests were updated consistently. Right now, each of the three adds the `annotations` field differently, making the reader wonder if there is any difference.